fundraising/banner.config.js

```diff
@@ -8,6 +8,11 @@
 	'use strict';
 	
 	Banner.config = {
+		encryption: {
+			libUrl: '../fundraising/openpgp.min.js',
+			publicKey: '',
+			elementId: ''
```

Comment:
I am wondering how the `elementId` is intended to be used. I have seen it being used in the test but it still not clear why it should be put in the config. The jquery element getting encrypted text is passed to the `encrypt` function so this is not intended for this, is it?